### PR TITLE
refactor: force symfony 5 bundle structure

### PR DESCRIPTION
**Pull request by lukadschaak**

### CODE REVIEW

**Review by nehlsen — APPROVED** [review]

File: tests/Integration/ServiceCompilerTest.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+namespace Neusta\Pimcore\PresentationBundle\Tests\Integration;
+
+use Pimcore\Test\KernelTestCase;
+
+class ServiceCompilerTest extends KernelTestCase
+{
+//    protected function setUp(): void
+//    {
+//        self::bootKernel(['environment' => 'test']);
+//    }
```
Comment (jdreesen) [inline tests/Integration/ServiceCompilerTest.php]: We shouldn't commit code that's commented out.

File: tests/app/TestKernel.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+use Neusta\Pimcore\PresentationBundle\NeustaPimcorePresentationBundle;
+use Pimcore\HttpKernel\BundleCollection\BundleCollection;
+use Pimcore\Kernel;
+
+/**
+ * Adapted by https://github.com/bernardosecades/tips-and-tricks/tree/master/functional-test-standalone-bundle
+ * in the path Tests/AppKernel.php.
+ */
```
Comment (jdreesen) [inline tests/app/TestKernel.php]: Is this still true?

File: tests/bootstrap.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+$kernelClass = 'TestKernel';
+$_SERVER['KERNEL_CLASS'] = $kernelClass;
+$_ENV['KERNEL_CLASS'] = $kernelClass;
+putenv('KERNEL_CLASS=' . $kernelClass);
+
+$appEnv = 'test';
+$_SERVER['APP_ENV'] = $appEnv;
+$_ENV['APP_ENV'] = $appEnv;
+putenv('APP_ENV=' . $appEnv);
```
Comment (jdreesen) [inline tests/bootstrap.php]: Not sure if we actually need to set it everywhere (probably just `$_SERVER` is enough), but if we do, I'd create a little helper like:

```suggestion
$setEnv = function (string $name, string $value): void {
    putenv("{$name}=".$_ENV[$name] = $_SERVER[$name] = $value);
};

$setEnv('PIMCORE_PROJECT_ROOT', __DIR__ . '/app');
$setEnv('KERNEL_CLASS', 'TestKernel');
$setEnv('APP_ENV', 'test');
```

Comment (lukadschaak) [reply tests/bootstrap.php]: You are right. I just could copy the helper function, you used before :)

Comment (jdreesen) [reply tests/bootstrap.php]: That's what I did in the comment :wink:

File: tests/app/TestKernel.php (hunk 2 above)
Comment (lukadschaak) [reply tests/app/TestKernel.php]: deleted as we know use a different implementation

File: tests/Integration/ServiceCompilerTest.php [hunk 4]
```diff
@@ -6,16 +6,11 @@
...
-//    {
-//        self::bootKernel(['environment' => 'test']);
-//    }
-
     /** @test */
     public function symfonyServiceDefinitionsMustCompile(): void
     {
-        // when this test passed, it means that the kernel could be loaded and there are no compiling errors in the
-        // symfony service definitions.
+        // when this test passed, it means that the kernel could be loaded (boostrap.php) and there are no
```
Comment (jdreesen) [inline tests/Integration/ServiceCompilerTest.php:12]: The kernel is not "loaded" (do you mean "booted"?) in the `boostrap.php`, it's only bootstrapped ;)

Comment (lukadschaak) [reply tests/Integration/ServiceCompilerTest.php:12]: I think you are right :D

**Pull request merged**